src/RecyclableMemoryStream.cs

```diff
@@ -475,6 +474,48 @@ public override byte[] ToArray()
 #pragma warning restore CS0809
 
         /// <summary>
+        /// Returns information about the currently used <c>byte[]</c> block and the amount of data written to it.
+        /// </summary>
+        /// <param name="block">
+        /// Assigned to the <see cref="ArraySegment{byte}"/> that holds a reference to the current <c>byte[]</c>
+        /// block and data length in it (<see cref="ArraySegment{byte}.Count"/>).
+        /// </param>
+        /// <returns>
+        /// <c>true</c> if the block at the current position exists and was retrieved;
+        /// <c>false</c> if the current instance is disposed.
+        /// </returns>
```

Comment:
Need more details about the behavior when the stream is full, and when the large buffer is used.